modules/slider/main/index.coffee

```diff
@@ -62,9 +60,10 @@ slide = (e) ->
     _.dragging = 'value'
     val.value = posToValue.call this, pos
 
+  prevValue = @value()
   @value(val)
-
-  @emit 'change', @value()
+  if prevValue isnt @value()
+    @emit 'change', @value()
   return
```

Comment:
This is why that's not the case:
```coffeescript
a = (a) ->
  return

a = (a) ->
  undefined
```
```js
var a;

a = function(a) {};

a = function(a) {
  return void 0;
};
```

Having an empty return statement is the correct way of returning nothing in coffeescript.